code/modules/modular_computers/file_system/programs/civilian/cargo_order.dm

```diff
@@ -5,7 +5,7 @@
 	size = 10
 	requires_ntnet = TRUE
 	available_on_ntnet = TRUE
-	usage_flags = PROGRAM_LAPTOP | PROGRAM_TELESCREEN | PROGRAM_CONSOLE
+	processor_strength_needed = PROCESSOR_MEDIUM
```

Comment:
I wouldnt mind if the processor requirement of that is decreased to the default (as that application can only be used to place orders, but not be used to approve them or call the shuttle)